goat_launch/params/locomotor_params.yaml

```diff
@@ -0,0 +1,29 @@
+robot_base_frame: base_link
+use_latest_pose: true
+global_costmap_class: nav_core_adapter::CostmapAdapter
+local_costmap_class: nav_core_adapter::CostmapAdapter
+global_costmap:
+  plugins: {type: "spatio_temporal_voxel_layer/SpatioTemporalVoxelLayer", name: global_rgbd_obstacle_layer}
+  global:
+    potential_calculator: dlux_plugins::AStar
+    traceback: dlux_plugins::GradientPath
+  path_caching: false
+  improvement_threshold: -1.0
+  publish_potential: false
+  print_statistics: false
+global_planner:
+  global:
+    plugin: {type: "nav_core2::GlobalPlanner", name: global}
+local_costmap:
+  update_costmap_before_planning: true
+  prune_plan: true
+  short_circuit_trajectory_evaluation: true
+  debug_trajectory_details: false
+  trajectory_generator_name: base_local_planner
+  goal_checker_name: dwb_plugins::SimpleGoalChecker
+  plugins: {type: "spatio_temporal_voxel_layer/SpatioTemporalVoxelLayer", name: local_rgbd_obstacle_layer}
+  local_rgbd_obstacle_layer:
+    voxel_decay: 15   # seconds if linear, e^n if exponential
+local_planner:
+  local:
+    plugin: {type: "nav_core2::LocalPlanner", name: local}
```

Comment:
this parameter should be locomotor/plugins, then you put parameters for this local planner into locomotor/local